packages/expect/src/__tests__/isError.test.ts

```diff
@@ -13,14 +13,14 @@ import {isError} from '../utils';
 
 // Copied from https://github.com/graingert/angular.js/blob/a43574052e9775cbc1d7dd8a086752c979b0f020/test/AngularSpec.js#L1883
 describe('isError', () => {
-  function testErrorFromDifferentContext(createError) {
+  function testErrorFromDifferentContext(createError: Function) {
```

Comment:
The `Function` type should be avoided, because it doesn't do any checking at call time, and accepts "functions" like constructors (which will error if not called with `new`).

It's not blocking yet, but `@typescript-eslint` added `Function` to the recommended types for `ban-types` in v3, so the less new usages of the `Function` type are added the codebase, the less work I have to do for #10177 :)

You should at worse be able to use `(...unknown[]) => any` as a quick way to make linting happy.

Reply:
Fixed.

Let me know if you have other blockers for the eslint migration.

Reply:
Awesome, thanks will do!